### PR TITLE
Adds APIM pages docs

### PR DESCRIPTION
**Pull request by tishan89**
## Purpose
Adds alert config docs
Adds overview page docs

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
-!!! Note
-    Content to be added. WIP.
\ No newline at end of file
+# Alerts Page
+This page illustrates information related to business alerts issued by the system for your currently selected
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md:2]: Can we explain, 'system' and 'whole team'?

Suggestion:
"This page shows the information related to business alerts issued by the system for your currently selected
 environment and organization. You can use this page as a health monitoring dashboard and make it visible to 
the whole team. You can use this page to drill down on each alert and discover possible anomalies or errors in your published APIs. 

See the topics given below to get details of the available widgets and understand how to use them to best effect."

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+ environment and organization. You can use this page as a health monitoring dashboard and make it visible to whole
+  team. Using this page you can drilldown on each alert and discover possible anomalies or errors in your published
+   APIs. Following sections of the document will guide you through the available widgets and how to use them to best
+    effect.
+    
+## Prerequisites
+By default, API analytics doesn't have any alerts enabled. Hence for you to use this page you should configure alerts
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md:9]:
```suggestion
Be sure to configure alerts if you want to use this page. By default, alerts are not enabled for API analytics. 
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md [hunk 3]
```diff
@@ -1,2 +1,13 @@
...
+ and then should have API invocations which will trigger alerts.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md:10]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 4]
```diff
@@ -1,9 +1,16 @@
 # Analytics Pages Introduction
-The new cloud analytics portal consists of several pages where pages are divided based on different stats (e.g., traffic
-, latency). There are a set of common controls that enable you to govern the data shown in pages. Each page will refer these
- common controls and page specific controls to visualize the information requested by the user.
+The new cloud analytics portal consists of several pages where pages are divided based on different functional aspects.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:2]:
```suggestion
The new cloud analytics portal consists of several pages, which are divided based on different functional aspects.
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 5]
```diff
@@ -1,9 +1,16 @@
...
+ (e.g., traffic, latency). Using these pages you can get the complete business analytics on your API management
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:3]:
```suggestion
You can use these pages to get the complete business analytics on your API management system. 

See the topics given below to get an overview of the main capabilities of **Analytics Pages**.
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 6]
```diff
@@ -1,9 +1,16 @@
...
+  system. This page will serve as the entry point on how to use the cloud analytics portal and educate you on how to
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:4]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 7]
```diff
@@ -1,9 +1,16 @@
...
+   get maximum use out of it.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:5]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 8]
```diff
@@ -1,9 +1,16 @@
 # Analytics Pages Introduction
-The new cloud analytics portal consists of several pages where pages are divided based on different stats (e.g., traffic
-, latency). There are a set of common controls that enable you to govern the data shown in pages. Each page will refer these
- common controls and page specific controls to visualize the information requested by the user.
+The new cloud analytics portal consists of several pages where pages are divided based on different functional aspects.
+ (e.g., traffic, latency). Using these pages you can get the complete business analytics on your API management
+  system. This page will serve as the entry point on how to use the cloud analytics portal and educate you on how to
+   get maximum use out of it. 
+   
+We will start off with introducing you to common controls which will govern the pages. Also we will provide an index
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:7]:
```suggestion
Let's start with the common controls that govern Analytics pages. See the related links in each topic given below for more details.
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 9]
```diff
@@ -1,9 +1,16 @@
...
+ to all pages that are present in the solution so that you can refer those pages in detail.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:8]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 10]
```diff
@@ -1,9 +1,16 @@
...
+The new cloud analytics portal consists of several pages where pages are divided based on different functional aspects.
+ (e.g., traffic, latency). Using these pages you can get the complete business analytics on your API management
+  system. This page will serve as the entry point on how to use the cloud analytics portal and educate you on how to
+   get maximum use out of it. 
+   
+We will start off with introducing you to common controls which will govern the pages. Also we will provide an index
+ to all pages that are present in the solution so that you can refer those pages in detail.   
  
 ## Common Controls
+ These are the set of common controls that enable you to govern the data shown in pages. Each page will refer these
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:11]:
```suggestion
These are the common controls that enable you to govern the data shown in Analytics pages. Each page uses these common controls and page-specific controls to visualize the information requested by the user. 

Explained below are these common controls and their usage.
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 11]
```diff
@@ -1,9 +1,16 @@
...
+ common controls and page specific controls to visualize the information requested by the user. Following sections
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:12]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 12]
```diff
@@ -1,9 +1,16 @@
...
+  will in detail explain each of these common controls and their usage.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:13]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 13]
```diff
@@ -92,9 +99,10 @@ This page displays information about OS platforms and HTTP Agents end users are
   
 ### Alerts Page
 
-This page illustrates information related to alerts issued by the system. You can use this page as a health monitoring
- dashboard and make it visible to whole team. Using the page you can get more information about issues alerts. This page
-  will serve as the starting point for all issued alerts. For more information please refer 
+This page illustrates information related to business alerts issued by the system for your currently selected
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:102]:
```suggestion
This page shows the information related to business alerts issued by the system for your currently selected
environment and organization. You can use this page as a health monitoring dashboard and make it visible to
the whole team. You can use this page to drill down on each alert and discover possible anomalies or errors in your published APIs.

For more information, see [Alerts Page]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts)
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 14]
```diff
@@ -92,9 +99,10 @@ This page displays information about OS platforms and HTTP Agents end users are
...
+ environment and organization. You can use this page as a health monitoring dashboard and make it visible to whole
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:103]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 15]
```diff
@@ -92,9 +99,10 @@ This page displays information about OS platforms and HTTP Agents end users are
...
+  team. Using this page you can drilldown on each alert and discover possible anomalies or errors in your published
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:104]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 16]
```diff
@@ -92,9 +99,10 @@ This page displays information about OS platforms and HTTP Agents end users are
...
+   APIs. For more information please refer
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:105]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md [hunk 17]
```diff
@@ -92,9 +99,10 @@ This page displays information about OS platforms and HTTP Agents end users are
...
 ### Alerts Page
 
-This page illustrates information related to alerts issued by the system. You can use this page as a health monitoring
- dashboard and make it visible to whole team. Using the page you can get more information about issues alerts. This page
-  will serve as the starting point for all issued alerts. For more information please refer 
+This page illustrates information related to business alerts issued by the system for your currently selected
+ environment and organization. You can use this page as a health monitoring dashboard and make it visible to whole
+  team. Using this page you can drilldown on each alert and discover possible anomalies or errors in your published
+   APIs. For more information please refer 
   [this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts) documentation.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md:106]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 18]
```diff
@@ -1,2 +1,38 @@
-!!! Note
-    Content to be added. WIP.
\ No newline at end of file
+#Overview Page
+![overview page full]({{base_path}}/assets/img/observe/overview/overview-page-full.png)
+This is the welcome page of the analytics portal. This page will mainly display a quick overview of the whole
+ management system. Targeted audience for the page are managers and business users who need quick insight. Also this
+  page can be used as a dashboard displaying current system status.
+ 
+## Page Walk Through
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:7]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 19]
```diff
@@ -1,2 +1,38 @@
...
+### Total Traffic Widget
+![overview page total traffic]({{base_path}}/assets/img/observe/overview/overview-page-total-traffic.png)
+
+Total traffic widget displays the total traffic your selected environment received during the selected time range
+. This will include both successful requests and error requests. If you want to investigate further on the traffic
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:12]:
```suggestion
. This includes both successful requests and error requests. If you want to investigate further on the traffic
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 20]
```diff
@@ -1,2 +1,38 @@
...
+ please move on to
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:13]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 21]
```diff
@@ -1,2 +1,38 @@
...
+  page can be used as a dashboard displaying current system status.
+ 
+## Page Walk Through
+### Total Traffic Widget
+![overview page total traffic]({{base_path}}/assets/img/observe/overview/overview-page-total-traffic.png)
+
+Total traffic widget displays the total traffic your selected environment received during the selected time range
+. This will include both successful requests and error requests. If you want to investigate further on the traffic
+ please move on to 
+ [traffic page]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-traffic).
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:14]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 22]
```diff
@@ -1,2 +1,38 @@
...
+ 
+### Latency Widget
+![overview page latency]({{base_path}}/assets/img/observe/overview/overview-page-latency.png)
+
+Latency widget displays the 95th percentile of all API latencies in your selected environment for the selected time
+ period. You can utilize this widget to know whether the whole system operates under given SLAs. This metric
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:20]:
```suggestion
 period. You can use this widget to know whether the whole system operates under given SLAs. This metric
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 23]
```diff
@@ -1,2 +1,38 @@
...
+  will give the first indication into whether there are any slow APIs. To investigate further please visit
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:21]:
```suggestion
 gives the first indication of slow APIs. To investigate further, see the 
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 24]
```diff
@@ -1,2 +1,38 @@
...
+
+Latency widget displays the 95th percentile of all API latencies in your selected environment for the selected time
+ period. You can utilize this widget to know whether the whole system operates under given SLAs. This metric
+  will give the first indication into whether there are any slow APIs. To investigate further please visit 
+  [latency page]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-latency).
+  
+## Error Rate Widget
+![overview page latency]({{base_path}}/assets/img/observe/overview/overview-page-error-rate.png)
+
+This widget will display the average error rate (error count/total request count) in your selected environment for
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:27]:
```suggestion
This widget displays the average error rate (error count/total request count) in your selected environment for
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 25]
```diff
@@ -1,2 +1,38 @@
...
+ the selected time period. You can utilize this widget as an indicator to know the health of the system. If error
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:28]:
```suggestion
 the selected time period. You can use this widget as an indicator to know the health of the system. If error
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 26]
```diff
@@ -1,2 +1,38 @@
...
+  rate is high you will need to investigate more via
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:29]:
```suggestion
  rate is high, investigate further using the 
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 27]
```diff
@@ -1,2 +1,38 @@
...
+
+This widget will display the average error rate (error count/total request count) in your selected environment for
+ the selected time period. You can utilize this widget as an indicator to know the health of the system. If error
+  rate is high you will need to investigate more via 
+  [error page]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-erros).
+  
+## API Request Summary Timeline
+![overview page latency]({{base_path}}/assets/img/observe/overview/overview-page-timeline.png)
+
+This chart combine above three widgets and plot them as a timeline. Left y-axis will show the 'request count' and 'error
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:35]:
```suggestion
This chart combines the above three widgets and plots them in a timeline. The y-axis shows the 'request count' and 'error
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 28]
```diff
@@ -1,2 +1,38 @@
...
+ count. x-axis displays time and right y-axis shows latency in milliseconds. Granularity of the data points will be
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:36]:
```suggestion
 count. The x-axis displays time and right y-axis on the right shows latency in milliseconds. Granularity of the data points are
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 29]
```diff
@@ -1,2 +1,38 @@
...
+  decided based on the time range you have selected. Tooltip will provide the exact value of all three metrics
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:37]:
```suggestion
  decided based on the time range you have selected. The tooltip accurately provides the exact value of all three metrics.
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 30]
```diff
@@ -1,2 +1,38 @@
...
+   accurately.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:38]:
```suggestion
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md (hunk 1 above)
Comment (tishan89) [reply en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md]: fixed with cc3b796

File: en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md [hunk 31]
```diff
@@ -0,0 +1,131 @@
+# Working with Alerts
+![alert config page]({{base_path}}/assets/img/observe/alerts/alert-config-full.png)
+
+One of the most useful features in APIM cloud analytics is the **Alerting support**. New analytics solution allows
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md:4]: One of the most useful features in APIM cloud analytics is **Alerting**. You can configure several types of alerts and receive alerts either within the portal or as emails. These alerts allow you to proactively monitor your API ecosystem and take corrective measures for any abnormalities you find. 
		
You can configure alerts for each environment within your organization separately. Alerts are configured per API and you can also modify/remove already-added alert configurations. 
		
Alerts are subjected to a suppression policy to ensure you are not overwhelmed with duplicate alert notifications. For each alert rule that you configure, there is a 10-minute suppression window applied by default. The suppression interval is currently not configurable. Also, note that suppression works per the alert rule. 

See the details given below about different alert types and how they are configured.

File: en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md [hunk 32]
```diff
@@ -0,0 +1,131 @@
...
+ configurations will always be per API basis. Users can also modify/remove already added alerting configurations. 
+ 
+Alert issuance is subjected to a suppression policy so that users will not be overwhelmed with duplicate alert
+ notifications. For each alert rule that you configure, there will be a 10 minute suppression window applied by
+  default. Suppression interval is currently not configurable. Also, please note that suppression will work on a per
+   alert rule basis. As the document progresses we will talk about different alert types and how to configure them.
+    
+## Alert Types
+### Latency Alert 
+Latency alerts allows users to configure alerts if response latency of their APIs are greater than a predefined
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md:19]: Configure Latency alerts to identify if the response latency of APIs is greater than a predefined threshold. This is useful when you have APIs that should honor SLAs and when you want to know about slow APIs proactively. 

The following topics explain how to create, modify, and remove latency alerts.

Comment (tishan89) [reply en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md]: Fixed with 48dd54c

File: en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md (hunk 31 above)
Comment (tishan89) [reply en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md]: Fixed with d555707

File: en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md [hunk 33]
```diff
@@ -0,0 +1,131 @@
...
+ threshold. This is useful when you have APIs which should honor SLAs and to get to know about slow APIs
+  proactively. Following sections will discuss how to create, modify and remove latency alerts.
+  
+#### Create Latency Alert
+- Click on 'User Profile' icon on the top right-hand corner and select **Settings**.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md:19]: This is a procedure? If so, we need to use the numbered list.
------------

To configure a latency alerts:

1.	Click **User Profile** on the upper-right corner and select **Settings**.

	This takes you to the **Latency Alert Configuration** section by default. The first part of the UI shows the alert-creation controls and the latter half shows the existing alerts and modify/remove controls.

		 latency alert config

2.	Verify that you are in the correct 
		[organization]({{base_path}}/observe/api-manager-analytics/configure-analytics/working-with-organizations) and select
		 the desired [environment]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction). 

3.	Select the API for which you want to create the alert by typing the API name in the **API Name** field. 

	!!! Info
	     -	As you type, the APIs that match the text you enter are listed. You can select the desired API from there. 
	     - Note that only the APIs you have invoked at least once are populated here. For other APIs, you need to give the API name in the desired format as instructed in the UI.
		 api name dropdown

4.	Go to the next column and select the desired metric against which you want to evaluate the alert rule. 

	!!! Info
	     The dropdown shows all available options. If there are multiple options, you can select the desired metric. If there is only one
		 option to select, the dropdown is disabled.
		
	latency metric type

5.	Go to the next column and specify the threshold in milliseconds. 

	!!! Info
	    When the 95th percentile of the selected metric exceeds the threshold provided here, alerts are triggered.
		
	 latency threshold

6.	Click **Add** once all the required alert configurations are specified. 

When added successfully the alert is shown in the lower table.
latency success

File: en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md [hunk 34]
```diff
@@ -0,0 +1,131 @@
...
+ exceed the threshold provided here alerts will be triggered.
+ 
+  ![latency threshold]({{base_path}}/assets/img/observe/alerts/latency-threshold.png)
+- Once all required information is added you can add the alert configuration using **Add** button. 
+- When added successfully it will be shown in the lower table.
+  
+  ![latency success]({{base_path}}/assets/img/observe/alerts/latency-success.png)
+  
+#### Modify Latency Alerts
+- Click on 'User Profile' icon on the top right-hand corner and select **Settings**.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md:49]: To modify an already-created alert:

1.	Click **User Profile** on the upper-right corner and select **Settings**.

	This takes you to the **Latency Alert Configuration** section by default. The latter table shows the already configured alerts. 

2.	Select the alert you want to modify from the table and click the **Edit** option under **Action**. 
		 latency config modify

3.	Change the parameters as you desire and click **Submit**. 
4.	If you want to discard the changes, click the stop/remove option.

File: en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md [hunk 35]
```diff
@@ -0,0 +1,131 @@
...
+  ![latency alert config]({{base_path}}/assets/img/observe/alerts/alert-config-full.png)
+- Later table shows the already configured alerts. From the table select the alert you want to modify and click the edit
+ icon under **Action** column. It will make that row editable.
+ 
+  ![latency config modify]({{base_path}}/assets/img/observe/alerts/latency-config-modify.png)
+- Change the parameters as you desire and click submit icon. If you want to discard the changes click on the stop
+/remove icon.
+
+#### Remove Latency Alerts
+- Click on 'User Profile' icon on the top right-hand corner and select **Settings**.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md:61]: To remove an existing latency alert:

1.	Click **User Profile** on the upper-right corner and select **Settings**.

	This takes you to the **Latency Alert Configuration** section by default. The latter table shows the already configured alerts. 

	latency alert config

2.	Select the alert you want to remove from the table and click the **bint** option under **Action**. 

Your alert configuration is now removed.

File: en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md [hunk 36]
```diff
@@ -0,0 +1,131 @@
...
+- This will take you to **Latency Alert Configuration** section by default.
+  
+  ![latency alert config]({{base_path}}/assets/img/observe/alerts/alert-config-full.png)
+- Later table shows the already configured alerts. From the table select the alert you want to remove and click the
+ bin icon under **Action** column. Your alert configuration will be removed.
+ 
+ ### Traffic Alert 
+ Traffic alerts allows users to configure alerts if request count of their APIs are greater than a predefined
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md:69]:
```suggestion
Configure Traffic alerts to identify if the request count of APIs is greater than a predefined threshold. This is useful when you have APIs that are using backends with traffic limits or monetized backends that require your system to proactively scale depending on the incoming traffic. 

The following topics explain how to create, modify, and remove traffic alerts.
```

File: en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md [hunk 37]
```diff
@@ -0,0 +1,131 @@
...
+  threshold. This is useful when you have APIs which are using traffic limited backends or monetized backends
+   where you need to proactively scale depending on your incoming traffic. Following sections will discuss how to
+    create, modify and remove traffic alerts.
+   
+ #### Create Traffic Alert
+ - Click on 'User Profile' icon on the top right-hand corner and select **Settings**.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md:75]:
```suggestion
To create a new Traffic alert:

1.	Click User Profile on the upper-right corner and select Settings.

	This takes you to the Latency Alert Configuration section by default. 

2.	Go to the **Traffic** tab.  

	This takes you to the **Traffic Alert Configuration** section. The first part of the UI shows the alert-creation controls and the latter half shows the existing alerts and the modify/remove controls.

		 ![traffic alert config]({{base_path}}/assets/img/observe/alerts/traffic-alert-config-full.png)

3.	Verify that you are in the correct [organization]({{base_path}}/observe/api-manager-analytics/configure-analytics/working-with-organizations) and select the desired [environment]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction). 

4.	Select the API for which you want to create the alert by typing the API name in the API Name field.

	!!! Info
		- As you type, the APIs that match the text you enter are listed. You can select the desired API from there.
		- Note that only the APIs you have invoked at least once are populated here. For other APIs, you need to give the API name in the desired format as instructed in the UI.

		 ![api name dropdown]({{base_path}}/assets/img/observe/alerts/api-name-dropdown.png)

5.	Go to the next column and select the desired metric against which you want to evaluate the alert rule.

	!!! Info
	    The dropdown shows all available options. If there are multiple options, you can select the desired metric. If there is only one
option to select, the dropdown is disabled.
	
		 ![traffic metric type]({{base_path}}/assets/img/observe/alerts/traffic-metric-type.png)

6.	Specify the threshold in requests per minute.

	!!! Info
	     When the per-minute request count for the given API exceeds the given threshold, the alerts are triggered.

		 ![traffic threshold]({{base_path}}/assets/img/observe/alerts/traffic-threshold.png)

7.	Click Add once all the required alert configurations are specified.

When added successfully the alert is shown in the lower table.
		
 ![traffic success]({{base_path}}/assets/img/observe/alerts/traffic-success.png)
```

File: en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md [hunk 38]
```diff
@@ -0,0 +1,131 @@
...
+  given API exceeds the threshold provided here alerts will be triggered.
+  
+   ![traffic threshold]({{base_path}}/assets/img/observe/alerts/traffic-threshold.png)
+ - Once all required information is added you can add the alert configuration using **Add** button. 
+ - When added successfully it will be shown in the lower table.
+   
+   ![traffic success]({{base_path}}/assets/img/observe/alerts/traffic-success.png)
+   
+ #### Modify Traffic Alerts
+ - Click on 'User Profile' icon on the top right-hand corner and select **Settings**.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md:106]:
```suggestion
To modify an existing alert:

1.	Click **User Profile** on the upper-right corner and select **Settings**.	
2.	Go to the **Traffic** tab. 
	
	This takes you to the **Traffic Alert Configuration** section. The first part of the UI shows the alert-creation controls and the latter half shows the existing alerts and the modify/remove controls.

	![traffic config pic]({{base_path}}/assets/img/observe/alerts/traffic-alert-config-full.png)

2.	Select the alert you want to modify from the table and click the **Edit** option under **Action**. 
		 
	![traffic config editable]({{base_path}}/assets/img/observe/alerts/traffic-config-editable.png)

3.	Change the parameters as you desire and click **Submit**. 
4.	If you want to discard the changes, click the stop/remove option.
```

File: en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md [hunk 39]
```diff
@@ -0,0 +1,131 @@
...
+   ![traffic config pic]({{base_path}}/assets/img/observe/alerts/traffic-alert-config-full.png)
+ - Later table shows the already configured alerts. From the table select the alert you want to modify and click the edit
+  icon under **Action** column. It will make that row editable.
+  
+   ![traffic config editable]({{base_path}}/assets/img/observe/alerts/traffic-config-editable.png)
+ - Change the parameters as you desire and click submit icon. If you want to discard the changes click on the stop
+ /remove icon.
+ 
+ #### Remove Traffic Alerts
+ - Click on 'User Profile' icon on the top right-hand corner and select **Settings**.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md:119]:
```suggestion
To remove an existing Traffic alert:
	
1.	Click **User Profile** on the upper-right corner and select **Settings**.	

2.	Go to the **Traffic** tab. 
		
		This takes you to the **Traffic Alert Configuration** section. The first part of the UI shows the alert-creation controls and the latter half shows the existing alerts and the modify/remove controls.
	
		![traffic config pic]({{base_path}}/assets/img/observe/alerts/traffic-alert-config-full.png)

3.	Select the alert you want to remove from the table and click the bin option under Action.

Your alert configuration is now removed.
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 40]
```diff
@@ -1,2 +1,35 @@
-!!! Note
-    Content to be added. WIP.
\ No newline at end of file
+#Overview Page
+![overview page full]({{base_path}}/assets/img/observe/overview/overview-page-full.png)
+This is the welcome page of the analytics portal. This page will mainly display a quick overview of the whole
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:3]:
```suggestion
This is the welcome page of the analytics portal. This page gives you a quick overview of the whole
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md [hunk 41]
```diff
@@ -1,2 +1,14 @@
-!!! Note
-    Content to be added. WIP.
\ No newline at end of file
+# Alerts Page
+This page illustrates information related to business alerts issued by the APIM analytics for your currently selected
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md:2]:
```suggestion
This page shows information related to business alerts issued by APIM analytics for your currently selected
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 42]
```diff
@@ -1,2 +1,35 @@
-!!! Note
-    Content to be added. WIP.
\ No newline at end of file
+#Overview Page
+![overview page full]({{base_path}}/assets/img/observe/overview/overview-page-full.png)
+This is the welcome page of the analytics portal. This page gives you a quick overview of the whole
+ management system. Targeted audience for the page are managers and business users who need quick insight. Also this
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:4]:
```suggestion
 management system. Targeted audience for the page are managers and business users who need quick insight. Also, this
```

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md [hunk 43]
```diff
@@ -1,2 +1,35 @@
...
+  page can be used as a dashboard displaying current system status.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-overview.md:5]:
```suggestion
  page can be used as a dashboard to view the current system status.
```

**Pull request merged**